IFC4x3/Properties/f/FlameRetardantWireType_0smteSsUD7mPHMM2lHlTVl/Documentation.md

```diff
@@ -0,0 +1 @@
+The type of wires which are self-extinguishing fire.
```

Comment:
This description is a little bit hard to understand. Is it to indicate if the wire self-extinguishes fire and if that is the case which type of cable does this?

Reply:
Really unclear, I agree. Especially looking at the property name.